Flaky test - NewPendingTransactionAcceptanceTest

eg https://github.com/hyperledger/besu/actions/runs/8257926441/job/22589310520?pr=6716

```
> Task :acceptance-tests:tests:acceptanceTestNotPrivacy

NewPendingTransactionAcceptanceTest > subscriptionToMinerNodeMustReceiveEveryPublishEvent() FAILED
    org.awaitility.core.ConditionTimeoutException: Assertion condition defined as a org.hyperledger.besu.tests.acceptance.dsl.pubsub.WebSocket 
    Expected size: 3 but was: 4 in:
    [org.hyperledger.besu.tests.acceptance.dsl.pubsub.SubscriptionEvent@646b6069,
        org.hyperledger.besu.tests.acceptance.dsl.pubsub.SubscriptionEvent@787d4e12,
        org.hyperledger.besu.tests.acceptance.dsl.pubsub.SubscriptionEvent@2712856c,
        org.hyperledger.besu.tests.acceptance.dsl.pubsub.SubscriptionEvent@28e86d] within 1 minutes.
        at app//org.awaitility.core.ConditionAwaiter.await(ConditionAwaiter.java:167)
        at app//org.awaitility.core.AssertionCondition.await(AssertionCondition.java:119)
        at app//org.awaitility.core.AssertionCondition.await(AssertionCondition.java:31)
        at app//org.awaitility.core.ConditionFactory.until(ConditionFactory.java:985)
        at app//org.awaitility.core.ConditionFactory.untilAsserted(ConditionFactory.java:769)
        at app//org.hyperledger.besu.tests.acceptance.dsl.WaitUtils.waitFor(WaitUtils.java:31)
        at app//org.hyperledger.besu.tests.acceptance.dsl.pubsub.WebSocket.verifyTotalEventsReceived(WebSocket.java:55)
        at app//org.hyperledger.besu.tests.acceptance.pubsub.NewPendingTransactionAcceptanceTest.subscriptionToMinerNodeMustReceiveEveryPublishEvent(NewPendingTransactionAcceptanceTest.java:121)

        Caused by:
        java.lang.AssertionError: 
        Expected size: 3 but was: 4 in:
        [org.hyperledger.besu.tests.acceptance.dsl.pubsub.SubscriptionEvent@646b6069,
            org.hyperledger.besu.tests.acceptance.dsl.pubsub.SubscriptionEvent@787d4e12,
            org.hyperledger.besu.tests.acceptance.dsl.pubsub.SubscriptionEvent@2712856c,
            org.hyperledger.besu.tests.acceptance.dsl.pubsub.SubscriptionEvent@28e86d]
            at org.hyperledger.besu.tests.acceptance.dsl.pubsub.WebSocket.lambda$verifyTotalEventsReceived$0(WebSocket.java:56)
            at org.awaitility.core.AssertionCondition.lambda$new$0(AssertionCondition.java:53)
            at org.awaitility.core.ConditionAwaiter$ConditionPoller.call(ConditionAwaiter.java:248)
            at org.awaitility.core.ConditionAwaiter$ConditionPoller.call(ConditionAwaiter.java:235)
            at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
            at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
            at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
            at java.base/java.lang.Thread.run(Thread.java:840)

24 tests completed, 1 failed
```

--- a/acceptance-tests/tests/src/test/java/org/hyperledger/besu/tests/acceptance/jsonrpc/EthSendRawTransactionAcceptanceTest.java
+++ b/acceptance-tests/tests/src/test/java/org/hyperledger/besu/tests/acceptance/jsonrpc/EthSendRawTransactionAcceptanceTest.java
@@ -44,6 +44,10 @@ public class EthSendRawTransactionAcceptanceTest extends AcceptanceTestBase {
     strictNode = besu.createArchiveNode("strictNode", configureNode((true)));
     miningNode = besu.createMinerNode("strictMiningNode", configureNode((true)));
     cluster.start(lenientNode, strictNode, miningNode);
+    // verify all nodes are done syncing so the tx pool will be enabled
+    lenientNode.verify(eth.syncingStatus(false));
+    strictNode.verify(eth.syncingStatus(false));
+    miningNode.verify(eth.syncingStatus(false));
   }
 
   @Test

--- a/acceptance-tests/tests/src/test/java/org/hyperledger/besu/tests/acceptance/pubsub/NewPendingTransactionAcceptanceTest.java
+++ b/acceptance-tests/tests/src/test/java/org/hyperledger/besu/tests/acceptance/pubsub/NewPendingTransactionAcceptanceTest.java
@@ -44,6 +44,9 @@ public class NewPendingTransactionAcceptanceTest extends AcceptanceTestBase {
     accountOne = accounts.createAccount("account-one");
     minerWebSocket = new WebSocket(vertx, minerNode.getConfiguration());
     archiveWebSocket = new WebSocket(vertx, archiveNode.getConfiguration());
+    // verify all nodes are done syncing so the tx pool will be enabled
+    archiveNode.verify(eth.syncingStatus(false));
+    minerNode.verify(eth.syncingStatus(false));
   }
 
   @AfterEach

